### ISSUE TITLE
Waiting on mountpoints...

### ISSUE THREAD

**Issue by bifa2** [TechPerplexed/Gooby#217]
Hi,
I know omnistream is the new approach, but since I've failed to get that working on numerous occassions (both upgrade, and fresh install) I'm trying to run this gooby install into my next life 😅 Unfortunately, it's broken again and I've spent the last few hours trying everything I can find from the closed issues.

```
sudo systemctl daemon-reload
sudo systemctl stop mergerfs
sudo systemctl stop rclonefs
/bin/fusermount -uz /mnt/rclone
/bin/fusermount -uz /mnt/google
```
Outputs:
```
/bin/fusermount: entry for /mnt/rclone not found in /etc/mtab
/bin/fusermount: entry for /mnt/google not found in /etc/mtab
```

After this, `/mnt` only has local, rclone, uploads. I can delete rclone just fine. Running `rclean` just results in the waiting on mountpoints error again.

I can start rclonefs and mergerfs and `/mnt` will contain `google` and `rclone` but the contents of my Google Drive aren't there. However, they are if I try with `rclone lsd GDrive-Movies`:`

I've tried rebooting the server multiple times, uninstalling and reinstalling rclone with a reboot in the middle (and not) and still it won't get past the 'waiting on mountpoints' message.

```
sudo systemctl status mnt-google.mount
● mnt-google.mount - /mnt/google
     Loaded: loaded (/proc/self/mountinfo)
     Active: active (mounted) since Wed 2023-03-29 19:54:43 UTC; 1min 38s ago
      Where: /mnt/google
       What: uploads:local:rclone

● rclonefs.service - Mount Google Drive (Rclone)
     Loaded: loaded (/etc/systemd/system/rclonefs.service; enabled; vendor preset: enabled)
     Active: failed (Result: exit-code) since Wed 2023-03-29 19:54:45 UTC; 3min 18s ago
    Process: 3138 ExecStartPre=/bin/mkdir -p ${RCLONEMOUNT} (code=exited, status=0/SUCCESS)
    Process: 3139 ExecStart=/usr/bin/rclone mount --allow-other --buffer-size 0M --dir-cache-time 96h --gid ${USERID} --log-level INFO --log-file ${HOMEDIR}/logs/rclone.log --timeout 1h --tpslimit 10 --tpsli>
   Main PID: 3139 (code=exited, status=1/FAILURE)

Mar 29 19:54:45 virtual-insanity systemd[1]: rclonefs.service: Failed with result 'exit-code'.
Mar 29 19:54:45 virtual-insanity systemd[1]: rclonefs.service: Scheduled restart job, restart counter is at 5.
Mar 29 19:54:45 virtual-insanity systemd[1]: Stopped Mount Google Drive (Rclone).
Mar 29 19:54:45 virtual-insanity systemd[1]: rclonefs.service: Start request repeated too quickly.
Mar 29 19:54:45 virtual-insanity systemd[1]: rclonefs.service: Failed with result 'exit-code'.
Mar 29 19:54:45 virtual-insanity systemd[1]: Failed to start Mount Google Drive (Rclone).

```

Hope you can still help here but if not, I understand!

**Comment by bifa2** [comment]
Ok so `rclone.log` is full of:
```
2023/03/29 19:42:10 INFO  : Starting transaction limiter: max 10 transactions/s with burst 10
2023/03/29 19:42:10 INFO  : Using --user gooby --pass XXXX as authenticated user
2023/03/29 19:42:10 NOTICE: Serving remote control on http://127.0.0.1:5572/
2023/03/29 19:42:10 Fatal error: failed to mount FUSE fs: fusermount: exec: "fusermount3": executable file not found in $PATH
```

May be relevant: https://github.com/rclone/rclone/issues/6844

**Comment by TechPerplexed** [comment]
Ack, yes I was afraid of it happening - I'm sure that has something to do with the new FUSE setting and I'm no longer quite in the loop why that's happening. Would it work for you if you installed an older version of Rclone (or the beta version)?

That said, I'm surprised that you didn't get Omni to work. I saw an issue there (https://github.com/kelinger/OmniStream/issues/53) and I'm ashamed to see we haven't replied to that yet - is your issue related?

**Comment by bifa2** [comment]
Thanks for the ever so quick reply. I've managed to fix it after finding that rclone thread :)
```
sudo apt install fuse3
sudo systemctl stop rclonefs
sudo systemctl start rclonefs
```

As for Omni... I've avoided in part because your replies are instant here and so the service is A+++++, but also because of issues I had in https://github.com/kelinger/OmniStream/issues/39 and seeing others with similar problems. I've a few users who use my server so taking it down for a week of tinkering to try and get things working (and not lose my Plex library) is hard to do 😅 Thank you again, I'll close this and perhaps it'll help some of us legacy users!
